package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "polymath-core",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

Comment:
I prefer the bump of the version during the release.

Reply:
We have already bumped 2.1 version. It helps the dApp guys. I am fine with any though.

Reply:
Ok then leave as it is